### DISCUSSION TITLE
Save/load profiles

### DISCUSSION THREAD

**Discussion by Scalpos** [hrvach/deskhop#162] [Ideas]
The web interface to change parameters is great.
What about saving/loading multiple profiles ?
If saving them on the device isn't possible, a string available as copy/paste would do the job, rather than having to manually enter different values when required.
I often have to work with a main laptop and different other devices.

**Comment by hrvach** [comment]
It is certainly possible, I will put this into the roadmap and try to do some development soon. Currently it's super busy at work so trying to find some time...

**Comment by mcascone** [comment]
Since you can copy the `config.htm` off the device when it's in config mode, you should be able to save different configs that way. Of course, this would require renaming the files for backup, and then renaming them when re-loading to the device. 

Or, you could keep the original filename and store each one in a descriptively-named folder. That would probably be easier both for manual bookkeeping and for automation.
